components/base/ImageDownload.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import styled from "@emotion/styled";
+import Tooltip from "./Tooltip";
+import { GREY_2, WHITE_PRIMARY } from "../colors";
+import { ReactComponent as DownloadIcon } from "../../assets/images/component-icons/download-icon.svg";
+
+const Container = styled.div`
+  box-sizing: border-box;
+  border: 1px solid ${GREY_2};
+  background-color: ${WHITE_PRIMARY};
+`;
+
+const DownloadDiv = styled.div`
+  height: 40px;
+  box-sizing: border-box;
+  border-bottom: 1px solid ${GREY_2};
+`;
+
+const DownloadIconWrapper = styled.div`
+  float: right;
+  margin: 12px 12px 12px auto;
+`;
+
+const ImageDiv = styled.div`
+  padding: 12px 13px;
+`;
+
+type Props = {
+  children: string;
+  // downloadFileName is used when the downloaded image must have a specific filename.
+  // Example - 'test-document-upload-success.png'
+  downloadFileName?: string;
+  tooltipText: string;
+};
+
+function ImageDownload({ children, downloadFileName, tooltipText }: Props) {
+  return (
+    <Container>
+      <DownloadDiv>
+        <DownloadIconWrapper>
+          <Tooltip text={tooltipText} hasArrow position="bottom" distance={11}>
+            <a href={children} download={downloadFileName || ""}>
+              <DownloadIcon />
+            </a>
+          </Tooltip>
+        </DownloadIconWrapper>
+      </DownloadDiv>
+      <ImageDiv>
+        <img src={children} alt="" />
```

Comment:
Can we give this a `max-width: 100%` or something so it shrinks on smaller mobile screens?

Reply:
Good point! Didn't think about that 😬

Reply:
This [commit](https://github.com/Dwolla/dev-portal/pull/48/commits/6a8d1f2db59c9945e35f9aed498cadcb9599b596) should take care of that!